src/Illuminate/Foundation/Http/Middleware/TrimStrings.php

```diff
@@ -5,27 +5,91 @@
 class TrimStrings extends TransformsRequest
 {
     /**
-     * The attributes that should not be trimmed.
+     * The exact attributes that should not be trimmed.
      *
      * @var array
      */
     protected $except = [
         //
     ];
 
+    /**
+     * The regular expressions matching attributes that should not be trimmed.
+     *
+     * @var array
+     */
+    protected $exceptPattern = [
+        //
+    ];
+
+    /**
+     * The regular expression generated from the exceptPattern.
+     *
+     * @var string
+     */
+    protected $patternString;
+
+    public function __construct()
+    {
+        $this->generatePatternString();
+    }
+
+    /**
+     * Generate the regular expression pattern from the exceptPattern.
+     *
+     * @return void
+     */
+    protected function generatePatternString()
+    {
+        if (count($this->exceptPattern) < 1) {
```

Comment:
There is no need to do a return here, just assign the parameter if `count(...) >= 1`